modules/custom/social_media_system/social_media_system.install

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the social_media_system module.
+ */
+
+use Drupal\Core\Config\FileStorage;
+
+/**
+ * Add featured view mode to media system.
+ */
+function social_media_system_update_24001(): void {
```

Comment:
For distro we use hook updates based on OS version, so it should start with `12xxx` ;) 
But not sure if it will land in version 12.0 or 12.1

Reply:
@ribel Hi!
So, the number should be 12001?